app/components/NavLoginSignupCart.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import FlatButton from 'material-ui/FlatButton';
+import ShoppingCart from 'material-ui/svg-icons/action/shopping-cart';
+import AccountBox from 'material-ui/svg-icons/action/account-box';
+import IconButton from 'material-ui/IconButton';
+import {browserHistory, Link} from 'react-router';
+
+const isLoggedIn = true;
+
+const NavLoginSignupCart = () => {
+  return (
+    <div>
+      {({isLoggedIn})
```

Comment:
this will always evaluate to true not mater what the value of `isLoggedIn` is b/c it is being wrapped in `{}`, making it an object (objects are truthy). 

To fix remove the inner `{}`.